yt/utilities/lib/geometry_utils.pyx

```diff
@@ -1,6 +1,6 @@
 # distutils: libraries = STD_LIBS
-# distutils: extra_compile_args = OMP_ARGS
-# distutils: extra_link_args = OMP_ARGS
+# distutils: extra_compile_args = CPP14_FLAG OMP_ARGS
```

Comment:
I believe adding the CPP14_FLAG doesn't do anything unless you also specify the language, doesn't it?

```suggestion
# distutile: language = c++
# distutils: extra_compile_args = CPP14_FLAG OMP_ARGS
```

Reply:
careful there, you misspelled `distutils` in your suggestion :)